//package com.sincetimes.redwar.game.comnunicate.protocol;



message RseQueryGalaxyWindow
{
	optional int32	ret = 1;
	repeated GalaxyWindow galaxyWindow = 2;
	optional int32 topLeftCoorX = 3;
	optional int32 topLeftCoorY = 4;
	optional int32 bottomRightCoorX = 5;
	optional int32 bottomRightCoorY = 6;
}

message GalaxyWindow
{
	optional int32 planetsFree = 1;
	optional int32 planetsOccupied = 2;
	optional string sku = 3;
	optional int32  starId = 4;
	optional int32 type = 5;
	optional int32 name = 6;
}
